cellfinder/napari/train/train.py

```diff
@@ -142,6 +143,7 @@ def widget(
         if yaml_files[0] == Path.home():  # type: ignore
             show_info("Please select a YAML file for training")
         else:
+            logger.info("Starting training process...")
```

Comment:
I suggest we use https://napari.org/dev/api/napari.utils.notifications.html instead of the logger in the `napari` submodule.

So here, we would do `show_info` instead of `logger.info`.